packages/RichTextEditing-Tests.package/RichTextDocumentReadStream.class/instance/isBeginningOfStructure..st

```diff
@@ -0,0 +1,4 @@
+as yet unclassified
```

Comment:
classify,

also class should be renamed to something like DocumentReadStreamTests